src/adv.py

```diff
@@ -38,14 +39,39 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player(room['outside'])
 
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+while True: 
+    # Prints the current room name
+    print(player.current_room.name)
+
+    # Prints the current description (the textwrap module might be useful here).
+    print(player.current_room.description)
+
+    try:
+        # Waits for user input and decides what to do.
+        direction = input("\nEnter direction: ").lower()[0]
+    except:
```

Comment:
Wow, nice job finding the whole `try/except` thing! I should try using that too.